intuita-webview/src/fileExplorer/TreeView/index.tsx

```diff
@@ -145,6 +149,17 @@ const TreeView = ({ node, nodeIds, fileNodes, searchQuery }: Props) => {
 		};
 	}, []);
 
+	useEffect(() => {
+		if (focusedNodeId === null || !fileNodeIds.has(focusedNodeId)) {
+			return;
+		}
+
+		vscode.postMessage({
+			kind: 'webview.fileExplorer.selectFile',
```

Comment:
is it a command (`selectFile`) or an event (`fileSelected`)?

Reply:
Event. `fileSelected`. Thanks. Changed it.